### PR TITLE
Delete libRocket from repo. Add as submodule.

### PR DESCRIPTION
**Pull request by DolceTriade**

### CODE REVIEW

File: libRocket.cmake [hunk 1]
```diff
@@ -0,0 +1,549 @@
+# Create libRocket.
```
Comment (mbasaglia) [inline libRocket.cmake]: Any reason why this isn't the librocket repo?

Comment (DolceTriade) [inline libRocket.cmake]: THe librocket repo has the actual build file. I felt this was much simpler than doing External project and hardcoding library paths. This is geared towards unvanquished (omitting python stuff).

**Comment by Kangz** [comment]
LVGTM!

**Comment by DolceTriade** [comment]
I will fix this later today and merge.

**Comment by DolceTriade** [comment]
Cherry picked to avoid conflicts.

**Pull request closed without merging**